root/usr/share/nethesis/NethServer/Language/en/NethServer_Module_OpenVpnTunnels.php

```diff
@@ -47,3 +48,7 @@
 $L['Topology_p2p_label'] = "P2P - One server for each client, shared key (PSK) authentication";
 $L['LocalPeer_label'] = "Local P2P IP";
 $L['RemotePeer_label'] = "Remote P2P IP";
+$L['weak_label'] = 'Weak';
+$L['strong_label'] = 'Strong';
+$L['TlsVersionMin_label'] = 'Enforce a minimum TLS version';
+$L['Cipher_Not_Compatible'] = 'Only a cipher name ending by CBC is compatible with the P2P topology';
```

Comment:
The "name ending by" sounds too complex. Can we just write?

    P2P topology requires a CBC cipher

...assuming both "P2P" and "CBC" are comprehensible from the surrounding context.